fbu-app/RegisterViewController.m

```diff
@@ -0,0 +1,129 @@
+//
+//  RegisterViewController.m
+//  fbu-app
+//
+//  Created by lucjia on 7/16/19.
+//  Copyright © 2019 lucjia. All rights reserved.
+//
+
+#import "RegisterViewController.h"
+#import "Parse/Parse.h"
+
+@interface RegisterViewController ()
+
+@property (strong, nonatomic) UITextField *usernameField;
+@property (strong, nonatomic) UITextField *emailField;
+@property (strong, nonatomic) UITextField *passwordField;
+@property (strong, nonatomic) UIButton *registerButton;
+@property (strong, nonatomic) UILabel *registerLabel;
+
+@end
+
+@implementation RegisterViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Create username field
+    self.usernameField = [[UITextField alloc] initWithFrame: CGRectMake(80.0f, 240.0f, 215.0f, 30.0f)];
+    self.usernameField.delegate = self;
+    self.usernameField.borderStyle = UITextBorderStyleRoundedRect;
+    self.usernameField.placeholder = @"Username";
+    [self.view addSubview:self.usernameField];
+    
+    // Create email field
+    self.emailField = [[UITextField alloc] initWithFrame: CGRectMake(80.0f, 300.0f, 215.0f, 30.0f)];
```

Comment:
You can reuse so much of this code by making a method like this:

```
- (UITextField *)registrationStyledTextFieldWithOrigin:(CGPoint)origin {
  UITextField *textField = [[UITextField alloc] initWithFrame:CGRectMake(origin.x, origin.y, 215.0f, 30.0f)];
  textField.delegate = self;
  textField.borderStyle = UITextBorderStyleRoundedRect;
  return textField;
}
```

You should get into the practice of doing this sort of thing any time you see code that's repeated. It's an important programming technique and will make your life way easier.